torch/_inductor/lowering.py

```diff
@@ -4733,6 +4733,7 @@ def register_pointwise_numeric_ldf64(op):
 
 register_foreach_pointwise(aten._foreach_add.List, add, allow_alpha=True)
 register_foreach_pointwise(aten._foreach_add.Scalar, add, allow_alpha=True)
+register_foreach_pointwise(aten._foreach_add.Tensor, add, allow_alpha=True)
```

Comment:
If we are adding this registration can you add a test as well in test_foreach.py

Reply:
Do you mean for eager? If so, the test is already added in the earlier PR of this stack (where the overload is added)

Reply:
No I mean for inductor

Reply:
Done in #111600